src/components/EpisodeCardsList/EpisodeCardsList.spec.js

```diff
@@ -0,0 +1,10 @@
+import {render, screen} from '@testing-library/react'
+import EpisodeCardsList from './EpisodeCardsList'
+  
+describe('EpisodeCardsList', () => {
+    it('renders', () => {
+        render(<EpisodeCardsList />)
+        expect(screen.getByText('EpisodeCardsList')).toBeInTheDocument()
+    })
+})
+      
```

Comment:
I think, you could be more specific here as well 😉

Reply:
Yes i have not yet implemented the real test :)